lib/websock.ex

```diff
@@ -58,10 +58,14 @@ defmodule WebSock do
           | {:reply, term(), {opcode(), message()} | [{opcode(), message()}], state()}
           | {:ok, state()}
           | {:stop, term(), state()}
+          | {:stop, term(), close_reason(), state()}
```

Comment:
Shouldn't it be `close_detail`?

```suggestion
          | {:stop, term(), close_detail(), state()}
```

Reply:
Yup! Thanks for the catch!